builtin/aws/lambda/plugin.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package aws.lambda;
+
+option go_package = "waypoint/builtin/aws/lambda";
+
+message Deployment {
```

Comment:
Docs on these fields please (even one liners, i.e. `func_arn` is kind of obvious but `version` is not to me)